stack/__main__.py

```diff
@@ -787,6 +788,17 @@ def find_service_account(kind: str, access_level: str) -> Optional[str]:
             member=f'serviceAccount:{ANALYSIS_RUNNER_SERVICE_ACCOUNT}',
         )
 
+    for access_level, service_account in service_accounts['hail']:
+        # Give access to sample-metadata cloud run service
+        gcp.cloudrun.IamMember(
+            f'samplemetadata-service-account-{access_level}-accessor',
```

Comment:
Doesn't make a difference semantically, just checking if this was a typo.

```suggestion
            f'sample-metadata-service-account-{access_level}-accessor',
```

Reply:
Just going for the one block, but this is better.